src/openforms/authentication/utils.py

```diff
@@ -21,14 +21,19 @@ class FormAuth(BaseAuth):
     machtigen: Optional[dict]
 
 
-def store_auth_details(submission: Submission, form_auth: FormAuth) -> None:
+def store_auth_details(
+    submission: Submission, form_auth: FormAuth, attribute_hashed: bool = False
+) -> None:
     attribute = form_auth["attribute"]
 
     assert (
         attribute in AuthAttribute.values
     ), f"Unexpected auth attribute {attribute} specified"
```

Comment:
I'm not sure what its purpose was: if I remove it, no test fails. Without it, I guess the creating/updating of the `AuthInfo` would raise an error because the `attribute` is not one of the `choices` specified, so we wouldn't end up with invalid data). 
So I will just remove it?

Reply:
Does saving with invalid data raise an `IntegrityError` or other `DatabaseError`? Probably not.
And what did we expect happened with these AssertionErrors?
Savest change is to rewrite this to:
```
if attribute not in AuthAttribute.values:
    raise AssertionError( f"Unexpected auth attribute {attribute} specified")
```

but if no tests failed. :shrug: